apps/web/src/pages/api/frontend/bildungsraum-share.ts

```diff
@@ -50,11 +52,28 @@ export default async function handler(
         }
 
         if (uuid.__typename === 'Article') {
-          const articleDocument = parseDocumentString(
-            uuid.currentRevision.content
-          ) as EditorArticleDocument
-          const articleContent = articleDocument.state.content
-          respondWithContent(articleContent)
+          const { introduction, content } = (
+            parseDocumentString(uuid.currentRevision.content) as {
+              state: {
+                introduction: EditorArticleIntroductionDocument
+                content: EditorRowsDocument
+              }
+            }
+          ).state
```

Comment:
maybe something like that would work?
```suggestion
            parseDocumentString((uuid.currentRevision as EditorArticleDocument).content)).state
```

Reply:
Sadly not, because then the `state` property of `content` and `introduction` is `unknown`, which gives TS errors down the line.

Reply:
@hejtful Hmmm interesting. Another Idea in editor-plugins.ts
```
export interface EditorArticleDocument {
  plugin: EditorPluginType.Article
  state: PrettyStaticState<ArticlePluginState> & {
    introduction: EditorArticleIntroductionDocument
    content: EditorRowsDocument
  }
  id?: string
  serloContext?: {
    articleTitle?: string
  }
}
```

If you agree I can add it in another PR as well.